Puppetfile

```diff
@@ -1,14 +1,14 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'katello/candlepin',        :git => 'https://github.com/Katello/puppet-candlepin'
-mod 'katello/capsule',          :git => 'https://github.com/Katello/puppet-capsule'
-mod 'katello/certs',            :git => 'https://github.com/Katello/puppet-certs'
-mod 'katello/common',           :git => 'https://github.com/Katello/puppet-common'
-mod 'katello/crane',            :git => 'https://github.com/Katello/puppet-crane'
-mod 'katello/katello',          :git => 'https://github.com/Katello/puppet-katello'
-mod 'katello/pulp',             :git => 'https://github.com/Katello/puppet-pulp'
-mod 'katello/qpid',             :git => 'https://github.com/Katello/puppet-qpid'
-mod 'katello/service_wait',     :git => 'https://github.com/Katello/puppet-service_wait'
+mod 'katello/candlepin',        '1.0.1'
+mod 'katello/capsule',          '3.0.0' 
+mod 'katello/certs',            '0.6.0'
+mod 'katello/common',           '0.1.3'
+mod 'katello/crane',            '1.0.0'
+mod 'katello/katello',          '5.0.1'
+mod 'katello/pulp',             '3.3.1'
+mod 'katello/qpid',             '1.2.2' 
+mod 'katello/service_wait',     '0.1.2'
```

Comment:
Hm, when did we start pinning these so strictly? I see it in 3.0 too, but the way foreman does it, and we did it for previous releases is to pin to a minor.

e.g., https://github.com/Katello/katello-installer/blob/KATELLO-2.4/Puppetfile#L29-L41

It makes it easier for smaller bug fixes I think to just re-run librarian and pull in the latest patch release.